src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -533,8 +532,7 @@ class MatrixDelayedUpdateCUDA
     Value* rcopy_ptr      = rcopy.data();
     // This must be Ainv must be tofrom due to NonlocalEcpComponent and possibly
     // other modules assumptions about the state of psiMinv.
-    PRAGMA_OFFLOAD("omp target data map(always, to: phiV_ptr[:norb]) \
-                    map(always, tofrom: Ainv_ptr[:Ainv.size()]) \
+    PRAGMA_OFFLOAD("omp target data map(always, tofrom: Ainv_ptr[:Ainv.size()]) \
```

Comment:
making a reference to psiMinv_ doesn't accomplish anything other than making this code harder to read.  I assume this was done at some point to minimize diffs but its making this code less clear now. Its quite important that Ainv is actually state i.e. of `*this`

Reply:
I removed the reference and made the comment less confusing.